src/control/convert.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include "util/color/rgbcolor.h"
+#include "util/assert.h"
+
+namespace mixxx {
+
+namespace control {
+
+typedef double value_t;
```

Comment:
Why the typedef? Is this supposed to replace all occurrences of `double` for CO signals/slots? If not I'd remove this and and rename the functions below to `doubleFromRgbColor`, because this would be confusing IMHO.

Reply:
No, you don't need to use it.

I was aiming for clean code, at least within this header file. Hard-coding primitive types into the domain layer is usually not desirable. Type aliases don't provide any type safety, so this is just a zero-cost semantic abstraction on the source code level.

But if it is confusing in contrast to the large amount of legacy code I will remove it.

Reply:
These are not arbitrary _double_ values, they have a special semantic in the context of _control values_. The standalone function name _doubleFromRgbColor()_ is therefore slightly misleading. But if it is invoked as _control::doubleFromRgbColor()_ the correlation might be obvious enough.